modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -614,16 +671,34 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>2m</code>, <code>3h</code>, <code>5d</code>.'));
 		so.rmempty = true;
 
-		so = ss.option(form.Value, 'duid', _('DUID'));
+		so = ss.option(form.Value, 'duid',
+			_('DUID'),
+			_('The DHCPv6-DUID of this host.'));
 		so.datatype = 'and(rangelength(20,36),hexstring)';
 		Object.keys(duids).forEach(function(duid) {
 			so.value(duid, '%s (%s)'.format(duid, duids[duid].hostname || duids[duid].macaddr || duids[duid].ip6addr || '?'));
 		});
 
-		so = ss.option(form.Value, 'hostid', _('IPv6 suffix (hex)'));
+		so = ss.option(form.Value, 'hostid',
+			_('IPv6-Suffix (hex)'),
+			_('The IPv6 interface identifier (address suffix) as hexadecimal number (max. 8 chars).'));
+
+		so = ss.option(form.Value, 'tag',
+			_('Tag'),
+			_('Set the given tag for matching hosts.'));
```

Comment:
We could extend the help text with an example of what this option looks like

Reply:
Please extend the wiki with an example of what this looks like.....

Reply:
Manpage has:

 The set:<tag> construct sets the tag whenever this --dhcp-host directive is in use. This can be used to selectively send DHCP options just for this host. More than one tag can be set in a --dhcp-host directive (but not in other places where "set:<tag>" is allowed). When a host matches any --dhcp-host directive (or one implied by /etc/ethers) then the special tag "known" is set. This allows dnsmasq to be configured to ignore requests from unknown machines using --dhcp-ignore=tag:!known If the host matches only a --dhcp-host directive which cannot be used because it specifies an address on different subnet, the tag "known-othernet" is set.

The tag:<tag> construct filters which dhcp-host directives are used. Tagged directives are used in preference to untagged ones.

Reply:
Tag is basically any arbitrary string, with a few special ones, e.g. "known"